test/functional/wallet.py

```diff
@@ -101,11 +101,17 @@ def run_test(self):
         # Exercise locking of unspent outputs
         unspent_0 = self.nodes[2].listunspent()[0]
         unspent_0 = {"txid": unspent_0["txid"], "vout": unspent_0["vout"]}
+        assert_raises_jsonrpc(-8, "Invalid parameter, expected locked output", self.nodes[2].lockunspent, True, [unspent_0])
         self.nodes[2].lockunspent(False, [unspent_0])
+        assert_raises_jsonrpc(-8, "Invalid parameter, output already locked", self.nodes[2].lockunspent, False, [unspent_0])
         assert_raises_jsonrpc(-4, "Insufficient funds", self.nodes[2].sendtoaddress, self.nodes[2].getnewaddress(), 20)
         assert_equal([unspent_0], self.nodes[2].listlockunspent())
         self.nodes[2].lockunspent(True, [unspent_0])
         assert_equal(len(self.nodes[2].listlockunspent()), 0)
+        assert_raises_jsonrpc(-8, "Invalid parameter, unknown transaction",
+          self.nodes[2].lockunspent, False, [{"txid": "0000000000000000000000000000000000", "vout": 0}])
```

Comment:
I personally don't like this indentation style. It looks like something between a new line and a new code block.

Would you mind aligning this continuation line with the opening parens:

```python
    assert_raises_jsonrpc(-8, "Invalid parameter, unknown transaction",
                          self.nodes[2].lockunspent, False, [{"txid": "0000000000000000000000000000000000", "vout": 0}])
```

same below